src/components/Banners/DismissableBanner/DismissableBanner.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react"
+import { FaInfoCircle } from "react-icons/fa"
+import { Center, Container, Icon, Text } from "@chakra-ui/react"
+import { Meta, StoryObj } from "@storybook/react"
+
+import DismissableBanner from "."
+
+const meta = {
+  title: "Molecules / Navigation",
```

Comment:
@tutizaraz I'm sorry, I made an error in my suggestion here!

```suggestion
  title: "Molecules / Navigation / DismissableBanner",
```

Have to append the title with `DismissableBanner` so that the stories are placed in the correct folder.

Reply:
I thought about it, but I figured you know better :)